_progress-bar.scss

```diff
@@ -0,0 +1,252 @@
+// progress-bar
+// ===
+
+// Configurable Variables
+// ---
+
+$progress-bar__progress-color: black !default;
+$progress-bar__success-color: $success-color !default;
+$progress-bar__error-color: $error-color !default;
+$progress-bar__background-color: $neutral-50 !default;
+
+
+// Dependencies
+// ---
+
+// Override Font Awesome's default path
+$fa-font-path: '/bower_components/font-awesome/fonts';
```

Comment:
Not to mention that if you decide not to use font-awesome, then you're stuck with `@import font-awesome/...` still running when you don't need it.